test/functional/test_framework/util.py

```diff
@@ -474,6 +474,7 @@ def write_config(config_path, *, n, chain, extra_config="", disable_autoconnect=
         #  min_required_fds = MIN_CORE_FDS + MAX_ADDNODE_CONNECTIONS + nBind = 151 + 8 + 3 = 162;
         #  nMaxConnections = available_fds - min_required_fds = 256 - 161 = 94;
         f.write("maxconnections=94\n")
+        f.write("par=2\n")
```

Comment:
What about `min(2, os.cpu_count())` to avoid forcing a second thread when only one core is available.

Reply:
Is there a realistic case where only one core is available?

Reply:
Only the rpi zero comes to mind. It seems like a simple change anyway. I wouldn’t overthink it too much.

Reply:
Or maybe a dockerized container